implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -432,14 +467,22 @@ async fn spawn_background_node(
     )
     .await?;
 
+    let proj_path = if let Some(path) = cmd.project.clone() {
```

Comment:
Maybe it would make sense to refactor this code with [this one](https://github.com/build-trust/ockam/pull/4230/files#diff-afac108b9e4f2377c7932bef242708078b72d74836268ccc0ff4a09a24f67a53R121)?